tests/rptest/tests/partition_force_reconfiguration_test.py

```diff
@@ -502,6 +582,21 @@ def no_pending_force_reconfigurations():
                     self.redpanda._admin.await_stable_leader(
                         topic=topic.name,
                         partition=part,
-                        timeout_s=self.WAIT_TIMEOUT_S,
+                        timeout_s=self.default_timeout_sec,
                         backoff_s=2,
                         hosts=self._alive_nodes())
+        topic_hws_after_recovery = {}
+        for t in topics:
+            topic_hws_after_recovery[
+                t.name] = self.get_topic_partition_high_watermarks(t.name)
+
+        for t in topics:
+            for id, (initial_hw, final_hw) in enumerate(
+                    zip(initial_topic_hws[t.name],
+                        topic_hws_after_recovery[t.name])):
+                self.logger.info(
+                    f"partition {t}/{id} replicas initial high watermark: {initial_hw} final high watermark: {final_hw}"
+                )
+            if t.redpanda_remote_write or t.replication_factor == 3:
+                assert all(
+                    [f >= 0.8 * i for i, f in zip(initial_hw, final_hw)])
```

Comment:
1. Unless we'd like to indent these 3 lines one step out, we are only checking the last partition here.
2. I'm also not sure if we compare correspondent replicas or just random combinations of pre- and post-recovery.
3. Would it maybe also make sense to assure that f <= i?